.changeset/olive-mugs-sort.md

```diff
@@ -0,0 +1,5 @@
+---
+'@chainlink/bank-frick-adapter': patch
```

Comment:
Should be a `minor` update, since if a node op starts using a B64 encoded private key, they can't downgrade past that version.